lib/kennel/models/monitor.rb

```diff
@@ -81,6 +81,12 @@ def as_json
         }
 
         data[:id] = id if id
+        if renotify_interval != 0
+          statuses = ["alert"]
+          statuses << "no data" if notify_no_data
+          statuses << "warn" if warning
```

Comment:
best to use the generated has for these too, to avoid duplicate method calls (for example calling `warning` twice)
so basically move this whole block just above the `validate_json` call